.github/workflows/update-major-and-minor-version.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Update major and minor tags
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-major-minor-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Git config
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Tag and push new major and minor versions
+        run: | 
+          VERSION=${{ github.event.release.tag_name }}
+          MAJOR=$(echo ${VERSION} | cut -d '.' -f 1)
```

Comment:
if `VERSION` is mistyped as `v.1.0`, then major tag will be `v` and minor tag will be `v.1`

Reply:
`VERSION` is not a manually typed input to this workflow. This workflow is triggered only AFTER the release. If there's a mistype, it SHOULD be detected and fixed PRE-release.